src/cmd_prepare_emscripten.rs

```diff
@@ -0,0 +1,9 @@
+use emscripten::initialize_emscripten;
+use error::Error;
+
+pub fn command_prepare_emscripten< 'a >() -> Result< (), Error > {
+    match initialize_emscripten( false, true ) {
+        None => return Err( Error::BuildError ),
+        Some( _emscripten ) => return Ok( () ),
```

Comment:
1. Please add a new variant to the `Error` type, something like `Error::EmscriptenNotAvailable`. It's `description` can be something like `"prepackaged Emscripten is not available for this platform"`.
2. Return it on `None` case.